standard-cluster-services/src/main/java/org/terracotta/monitoring/PlatformService.java

```diff
@@ -30,4 +31,8 @@
    * Dumps platform state into logs
    */
   void dumpPlatformState();
+  /**
+   * Provides the raw config file from server startup
+   */
+  InputStream getPlatformConfiguration();
```

Comment:
I'd rather have the downstream consumer do the conversion.  We can do it prior but that just means possibility messing up the character set.  I believe some consumers (jaxb) have the ability to detect things like that if the file is not in the platforms default. That being said, I'm not opposed to doing the conversion to the os default if that is what is truly desired.